### ISSUE TITLE
Homepage and About Page Copy

### ISSUE THREAD

**Issue by allella** [codeforgreenville/hackgreenville-com#26]
The seed for a HG Purpose, Mission, Vision, Culture is up on the HG Slack and open for comments and review.

The proposal is copied below, but [please comment on the HG Slack thread](https://app.slack.com/client/T0420TN6E/C0420TN6U/thread/C042530GC-1576100889.021400?cdn_fallback=1
).

It was decided in the previous thread, per Lelia's suggestion, to set a HackGreenville purpose, mission, vision, and culture.  That will make it easier to write copy for the website and HG will have a guiding "About" page to further explain the why we exist, who we are, who we are not, what we do, and how we do it.

Taking the previous thread's comments into consideration, here's a summary and starting point:

**Purpose**
HackGreenville exists to support rewarding personal growth among the "hackers" of Greenville, SC and the surrounding area.

**Mission**
HackGreenville hosts an online community and discovery tools to promote local, non-commercial tech opportunities for learning, sharing, and connecting.

**Vision**
HackGreenville desires to become the GO TO (pun intended!) brand and resource for tech people discovering the area's "hacker" culture and opportunities.

**Culture / Norms / Guiding Principles**

- Be nice. Keep egos and biases in check.
- Have fun. Trolling is not cool.
- Pay-it-forward. You learned most of what you know from others.
- Give more than you take. This is not a community for selfish pursuits.

**Comment by allella** [comment]
We have an About page pretty well nailed down on the HG #general conversation
https://hackgreenville.slack.com/archives/C0420TN6U/p1576510576002000

This should help with copy for the homepage

## About HackGreenville
HackGreenville was started as a [Slack chat group](https://hackgreenville.com/join-slack) in March 2015 by Andrew Orr (@webdood) and Dave Brothers (@davebrothers).

The [Slack community](https://hackgreenville.com/join-slack) has stood the "test of time" and continues to be a daily, active hub for countless and varied conversations.

The HackGreenville.com website was initially developed by the [SC Codes](https://www.sccodes.org/) pilot program and later adopted and expanded by [Code For Greenville](https://codeforgreenville.org/). The site leverages [open data APIs developed by Code For Greenville](https://data.openupstate.org/) to promote local, related meetup groups and events.

The following summarizes the overall community's purpose, mission, vision, and culture. 

### Purpose
HackGreenville exists to foster personal growth among the "hackers" of Greenville, SC and the surrounding area.

### Mission
HackGreenville hosts an online community and discovery tools to promote local, non-commercial tech opportunities for learning, sharing, and connecting.

### Vision
HackGreenville desires to become the GO TO (pun intended!) brand and resource for tech people discovering the area's "hacker" culture and opportunities.

### Culture and Guiding Principles 

- We're open to all hackers, makers, and tinker types. We're not exclusive to software.
- Be nice. Keep egos and biases in check.
- Have fun. Trolling is not cool.
- Pay-it-forward. You learned most of what you know from others.
- Give more than you take. This is not a community for selfish pursuits.

**Comment by allella** [comment]
There's also "Where we do fun new things in a friendly smart environment." content in the footer that could use some love.

**Comment by jeremywight** [comment]
Updated the content further, though we could definitely add a few more images in the carousel with further content

**Comment by allella** [comment]
@zach2825 It wasn't immediately obvious how to showcase the Slack community and the meetup side with just one heading, so we removed the headings for now.

Jeremy had suggested we could either have the main hero image showcase snippets from the Slack community and the IRL side.

In the interest of time we figured we could do without the headings until we either had a photo that isn't so heavy on the in-person aspect.  Or, when there's a carousel then it could help showcase the different aspects.

I'd be fine without the carousel for this release, but if #12 makes it easy then that works too. Then, we'd still need another photo and possibly headings for the photos.

Also, Pamela mentioned an eventual secondary audience of showcasing the community for economic development / employer reasons, but we didn't come to much resolution on what that angle would entail for this release, if any.

Thoughts?

**Comment by allella** [comment]
@zach2825 It wasn't immediately obvious how to showcase the Slack community and the meetup side with just one heading, so we removed the headings for now.

@jeremywight  had suggested we could either have the main hero image showcase snippets from the Slack community and the IRL side.

In the interest of time we figured we could do without the headings until we either had a photo that isn't so heavy on the in-person aspect. Or, when there's a carousel then it could help showcase the different aspects.

I'd be fine without the carousel for this release, but if #12 makes it easy then that works too. Then, we'd still need another photo and possibly headings for the photos.

Also, @pamelawoodbrowne  mentioned an eventual secondary audience of showcasing the community for economic development / employer reasons, but we didn't come to much resolution on what that angle would entail for this release, if any.

Thoughts?
